src/jobs.rs

```diff
@@ -0,0 +1,62 @@
+//! Job system
+use std::sync::mpsc;
+use std::thread;
+use std::collections::hash_map::{HashMap, Entry};
+use std::cell::RefCell;
+
+struct Job {
+    rx: mpsc::Receiver<Output>,
+    handle: thread::JoinHandle<()>,
+}
+
+type Output = String;
+type JobId = String;
+
+const NO_RESULTS_YET: &str = "NO RESULTS YET";
+const NO_SUCH_JOB: &str = "NO SUCH JOB";
+const JOB_PANICKED: &str = "JOB PANICKED";

```

Comment:
You need to either change this to _have_ underscores, or change the defines above to _not_ have underscores. It's an error that still exists in my impl.

Reply:
Ill fix the defines so I dont have to rebuild RUSTG in my other PR